src/chesster.js

```diff
@@ -830,22 +831,22 @@ function validateUserResult(details, result){
         valid: true,
         reason: ""
     };
-    if( details.winner && result.result == "1/2-1/2" ){
+    if( details.winner && result.result === "1/2-1/2" ){
```

Comment:
This is probably fine - but this is a case where I think _.isEqual is probably better.